snippets/examples-cards.jsx

```diff
@@ -0,0 +1,50 @@
+export const ExamplesCards = ({
+  examples,
+  runtime,
+}) => {
+  const examplesData = {
```

Comment:
I know this should be in its own JSON or at least its own JS file. Unfortunately, Mintlify doesn't allow you to put it anywhere but inside of the component that's going to use it.